Apps/Sandcastle/gallery/development/Picking Sublayer Regions.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Use Viewer to start building new applications or easily embed Cesium into existing applications.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases">
+    <title>Cesium Demo - Picking Sublayer Regions</title>
```

Comment:
Please don't hand-edit Sandcastle's save files.  Use the "Save As..." button in Sandcastle, so that if a user modifies it inside Sandcastle and then re-exports it, there will be no diffs other than what the user did inside of Sandcastle itself.